services/player/api/serializers.py

```diff
@@ -13,15 +13,23 @@ class Meta:
         model = Player
         fields = ('username',)
 
+
 class FirstNameSerializer(serializers.ModelSerializer):
     class Meta:
         model = Player
         fields = ('first_name',)
-        
+
+
 class LastNameSerializer(serializers.ModelSerializer):
     class Meta:
         model = Player
         fields = ('last_name',)
+
```

Comment:
unnecessary new line

Reply:
The same thing here, two new lines are correct.